backend/coins/btc/account.go

```diff
@@ -549,6 +554,9 @@ func (account *Account) subscribeAddress(
 
 // Transactions wraps transaction.Transactions.Transactions()
 func (account *Account) Transactions() []accounts.Transaction {
+	if account.fatalError {
+		return []accounts.Transaction{}
```

Comment:
Dito. Do we call/need this? Or could we just panic (as part of the contract)?

Reply:
done